<launch>
  <node pkg="rhbp_core" type="topic_listener.py" name ="TopicListenerNode" />
</launch>
